### PR TITLE
Bugfix/filepath multiple worker

### PR DESCRIPTION
**Pull request by veseln**
Adds fail_on_missing flag to ExportToTiff task which determines whether
the task should fail if an export of a missing feature is requested.

If fail_on_missing is set to True -  a ValueError will be raised when trying to Export a feature that is not present in the eopatch.

If fail_on_missing is set to False - only a warning will be logged

By default, the flag is set to True.

### CODE REVIEW

File: io/eolearn/io/local_io.py [hunk 1]
```diff
@@ -308,4 +324,4 @@ def execute(self, eopatch=None, *, filename=None):
 
         eopatch[feature_type][feature_name] = data
 
-        return eopatch
+        return eopatch
```
Comment (devisperessutti) [inline io/eolearn/io/local_io.py]: can you add a newline here for pylint

**Comment by devisperessutti** [comment]
good for me. Travis fails for Python 3.6 for some reason (seems a bit flaky)

**Comment by veseln** [comment]
@AleksMat  Can you have a look? If it's ok, you can merge.

File: io/eolearn/io/local_io.py [hunk 2]
```diff
@@ -158,10 +162,22 @@ def execute(self, eopatch, *, filename=None):
         :param filename: filename of tiff file or None if entire path has already been specified in `folder` parameter
             of task initialization.
         :type filename: str or None
+        :param fail_on_missing: should the pipeline fail if a feature is missing or just log warning and return
+        :type fail_on_missing: bool
+
```
[the remark this answers is not in the document]
Comment (veseln) [reply io/eolearn/io/local_io.py]: Ah yes, had the parameters in the function first and later moved it to init. Forgot to move the documentation also. Thanks.

[the remark this answers is not in the document]
Comment (AleksMat) [reply io/eolearn/io/local_io.py]: Sorry for complicating, but could you also delete line 167 (docs strings)

**Pull request merged**